.gitignore

```diff
@@ -12,7 +12,13 @@ api_keys.py
 final_jsons/*.json
 shape_files/*.json
 *.csv
+*.xls
 *.xlsx
+!data_folder/*.xlsx
+!data_folder/*.csv
```

Comment:
We want to avoid adding large data files. You can manually add files we want to commit.

Reply:
So something like this?

```
!data_folder/food_insecurity_rates_12.15.2020.xlsx
!tests/resources/test_food_insecurity_xlsx.xlsx
!tests/resources/test_food_insecurity_xls.xls
!tests/resources/test_food_insecurity_csv.csv
```

Reply:
You can manually add the files to be tracked.

Reply:
With `git add <file_name>`

Reply:
I've backed out the exception rules form the .gitignore.